lib/ansible/modules/cloud/docker/docker_host_facts.py

```diff
@@ -0,0 +1,325 @@
+       module has been superseded by L(docker,https://pypi.org/project/docker/)
+       (see L(here,https://github.com/docker/docker-py/issues/1310) for details).
+       For Python 2.6, C(docker-py) must be used. Otherwise, it is recommended to
+       install the C(docker) Python module. Note that both modules should I(not)
+       be installed at the same time. Also note that when both modules are installed
+       and one of them is uninstalled, the other might no longer function and a
+       reinstall of it is required."
+    - "Docker API >= 1.21"
+'''
+
+EXAMPLES = '''
+- name: Get info on docker host
+  docker_host_facts:
+  register: result
+
+- name: Get info on docker host and list images
+  docker_host_facts:
+    images: yes
+  register: result
+
+- name: Get info on docker host and list images matching the filter
+  docker_host_facts:
+    images: yes
+    images_filters:
+      label: "mylabel"
+  register: result
+
+- name: Get info on docker host and verbose list images
+  docker_host_facts:
+    images: yes
+    verbose_output: yes
+  register: result
+
+- name: Get info on docker host and used disk space
+  docker_host_facts:
+    disk_usage: yes
+  register: result
+
+- debug:
+    var: result.docker_host_facts
+
+'''
+
+RETURN = '''
+docker_host_facts:
+    description:
+      - Facts representing the basic state of the docker host. Matches the C(docker system info) output.
+    returned: always
+    type: dict
+docker_volumes_list:
+    description:
+      - List of dict objects containing the basic information about each volume.
+        Keys matches the C(docker volume ls) output unless I(verbose_output=yes).
+        See description for I(verbose_output).
+    returned: When I(volumes) is C(yes)
+    type: dict
+docker_networks_list:
+    description:
+      - List of dict objects containing the basic information about each network.
+        Keys matches the C(docker network ls) output unless I(verbose_output=yes).
+        See description for I(verbose_output).
+    returned: When I(networks) is C(yes)
+    type: dict
+docker_containers_list:
+    description:
+      - List of dict objects containing the basic information about each container.
+        Keys matches the C(docker container ls) output unless I(verbose_output=yes).
+        See description for I(verbose_output).
+    returned: When I(containers) is C(yes)
+    type: dict
+docker_images_list:
+    description:
+      - List of dict objects containing the basic information about each image.
+        Keys matches the C(docker image ls) output unless I(verbose_output=yes).
+        See description for I(verbose_output).
+    returned: When I(images) is C(yes)
+    type: dict
+docker_disk_usage:
+    description:
+      - Information on summary disk usage by images, containers and volumes on docker host
+        unless I(verbose_output=yes). See description for I(verbose_output).
+    returned: When I(disk_usage) is C(yes)
+    type: int
```

Comment:
Is this type still up-to-date?

Reply:
I was wondering if I should leave the value it returns by default or change it either to `complex` or to `int or dict` - the last one I am not sure if it would be allowed by the engine building the docs.

Reply:
I updated it so it is always `dict` now